marketplacetests/tests/test_marketplace_make_an_in_app_payment.py

```diff
@@ -11,38 +12,42 @@
 
 class TestMakeInAppPayment(MarketplaceGaiaTestCase):
 
-    APP_NAME = 'Testing In-App-Payments'
-    APP_TITLE = 'In-App-Payments'
-    PIN = '1234'
+    test_data = {
+        'app_name': 'Testing In-App-Payments',
+        'app_title': 'In-App-Payments',
+        'server': 'marketplace-dev.allizom.org',
```

Comment:
Will we ever need to change the server? If we did, would it need to be across all tests? I wonder if setting this elsewhere makes sense, either by a default value or an override in a central location.